.github/workflows/build.yml

```diff
@@ -123,10 +125,12 @@ jobs:
       with:
           python-version: '3.4.4'
           architecture: 'x86'
-    - name: Install VS libs
-      run: choco install vcexpress2010
+    - name: Upgrade pip
+      run: python -m pip install --upgrade pip==19.1.1
```

Comment:
I think you can lose the `--upgrade` flag here, since you're installing a **fixed** version (==19.1.1), aren't I right?